platform/macos/src/MGLMapView.mm

```diff
@@ -252,7 +252,7 @@ + (NSArray *)restorableStateKeyPaths {
     return @[@"camera", @"debugMask"];
 }
 
-- (void)commonInit {
+- (void)commonInit:(nullable NSString*)fontFamily {
```

Comment:
It needs to be possible to set the font family at an arbitrary time after creating the `mbgl::Renderer`. The reason this method doesn’t take any parameters – not even the view’s frame – is that we have no control over the parameters at initialization when the view is initialized from a XIB or storyboard.

Reply:
#10612 would address the immediate issue here pretty elegantly, although there would remain room for future improvement.

Reply:
:+1: